packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx

```diff
@@ -1059,6 +1116,9 @@ export class FieldEditor {
     const boolSupportInternalName = objProductConfig.editInternalName;
     const strBaseClassName = 'fw-field-editor';
     const objFieldBuilder = this.fieldBuilderOptions;
+
+    /** Adding extra check for status type */
+    const isStatusType = objFormValue.name === 'status';
```

Comment:
Add the name of the field in config as mentioned above and check if builder supports toggle feature

Reply:
check changed